src/commandline_options.hpp

```diff
@@ -164,6 +164,10 @@ class commandline_options
 	utils::optional<std::string> preprocess_path;
 	/** Target (output) path that was given to the --preprocess option. */
 	utils::optional<std::string> preprocess_target;
+	/** True if --preprocess-string was given on the command line. Starts Wesnoth in preprocessor-only mode. */
+	bool preprocess_string;
```

Comment:
What's the point of this when `preprocess_source_string` is an optional?

Reply:
dropped.